tests/test_a11y.py

```diff
@@ -167,6 +168,13 @@ def test_axe_core(
     # Wait for CSS transitions (Bootstrap's transitions are 300 ms)
     page.wait_for_timeout(301)
 
+    # On the PyData Library Styles page, wait for ipywidget to load and for our
+    # JavaScript to apply tabindex="0" before running Axe checker (to avoid
+    # false positives for scrollable-region-focusable).
```

Comment:
Interesting, I did not notice before that this was the case, i.e. the `tabindex` being applied later. I suppose it make sense since we are delegating this to JS.

Reply:
Yeah, you might have seen [my first tab stop implementation](https://github.com/pydata/pydata-sphinx-theme/pull/1636/files), which adds tabindex="0" at build time (via HTML created by Python), and might not have noticed [my second tab stop implementation](https://github.com/pydata/pydata-sphinx-theme/pull/1777), which adds a zero tab index at run time (via DOM manipulation by JavaScript). 

Fun fact. This change was driven by an article you shared (or maybe it was Smera) :sunflower:  [Inclusive Components - Data Tables - Only Focusable Where Scrollable](https://inclusive-components.design/data-tables/#onlyfocusablewherescrollable)